src/unit.rs

```diff
@@ -975,6 +976,8 @@ impl<'input, Endian> Attribute<'input, Endian>
     /// name.
     ///
     /// See "Figure 20. Attribute encodings" and "Figure 21. Attribute form encodings".
+    #[allow(cyclomatic_complexity)]
+    #[allow(match_same_arms)]
```

Comment:
Some of the match clauses have the same arms, so clippy wants them to be a single match clause joined by `|`. That isn't appropriate in this case, IMO, although usually it is.